two_factor/templates/two_factor/_wizard_actions.html

```diff
@@ -1,14 +1,21 @@
 {% load i18n %}
 
-<div class="button-row">
-  {% if cancel_url %}
-    <a href="{{ cancel_url }}" class="pull-right btn btn-link">{% trans "Cancel" %}</a>
-  {% endif %}
-  {% if wizard.steps.prev %}
-    <button name="wizard_goto_step" type="submit" value="{{ wizard.steps.prev }}" class="btn btn-default">{% trans "Back" %}</button>
-  {% else %}
-    <button disabled name="" type="button" class="btn btn-disabled">{% trans "Back" %}</button>
-  {% endif %}
-  {% comment %} <a href="{% url 'two_factor:setup' %}" class="btn btn-default">{% trans "Setup" %}</a> {% endcomment %}
-  <button type="submit" class="btn btn-primary">{% trans "Next" %}</button>
+{% if wizard.steps.current == 'auth' %}
```

Comment:
please, update only /two_factor/admin templates. This template is used outside of the admin

Reply:
I added a file _wizard_actions.html  in the core folder

Reply:
the _wizard_actions.html is used in the admin and in two files from the core folder, the core/phone_register.html and the core/setup.html files, core/setup.html is used to display this 
![image](https://user-images.githubusercontent.com/49318924/129335147-a5cc6a4b-60b4-4f2b-b737-cfddc9827d20.png)
I think it is a good thing to modify the _wizard_actions.html as every files which uses it need this modification, don't you think?

Reply:
Let's discuss what happened to this lib so far so see the whole picture:
* the original lib was created to add 2fa support to django app. It has boostrap styling in html templates. It doesn't support 2fa for django admin out of the box
* This forked was made to support 2fa in django admin including django admin styling in the templates.  To achieve it all templates were copied to `two_factor/admin` folder. 

The idea is that django admin 2fa views should use `two_factor/admin` templates, and other 2fa view should use templates from other folders. At least this is how I understand it by looking through commit history, @sergei-maertens can correct me. 

If `two_factor/_wizard_actions.html` is included in the admin templates this is a copy-paste error, which should be fixed. We have  `two_factor/admin/_wizard_actions.html` which should be used for admin views. 

Unfortunately, the templates in `two_factor/admin` folder have a lot of copy-paste errors, which leads to confusion. 
But to make our live easy we should stick to the idea, that django admin styling should be done in `two_factor/admin` folder. templates in other folders can be left with their bootstrap styling unchanged

Reply:
Correct @annashamray!

_All_ admin templates should be namespaced under the admin, including the wizard templates. This means there are two sets of templates: core and admin. A lot of duplication, but it's necessary.